src/components/ImageGalleryGrid/index.js

```diff
@@ -62,4 +64,14 @@ class ImageGalleryGrid extends React.Component {
     }
 }
 
+ImageGalleryGrid.propTypes = {
+    images: PropTypes.object,
+    user: PropTypes.object,
+    editor: PropTypes.object,
+    fetchUserImages: PropTypes.func,
+}
+
+const mapDispatchToProps = (dispatch) => ({
+    fetchUserImages: (user, amount) => dispatch(fetchUserImagesAction(user, amount)),
+})
 export default connect()(ImageGalleryGrid)
```

Comment:
Forgot to use `mapDispatchToProps` in `connect()`

Reply:
Lovely! Thanks